microscope/find_citations.py

```diff
@@ -75,7 +73,7 @@ def get_citations(
         # In this case, we're not sure yet what the citation's antecedent is.
         # It could be any of the previous citations above. Thus, like an Id.
         # citation, for safety we won't resolve this reference yet.
-        elif strip_punct(citation_token.lower()) == "supra":
+        elif re.match(r"[^a-z0-9]*supra[^a-z0-9]*$", citation_token.lower()):
```

Comment:
@mattdahl can you give your read on the change to this line? I don't have as good a sense of what the `supra` data looks like as you do.

Reply:
I think this should be fine!